spec/unit/type/yumrepo_spec.rb

```diff
@@ -292,6 +309,7 @@
 
     describe "priority" do
       it_behaves_like "a yumrepo parameter that can be absent", :priority
+      it_behaves_like "a yumrepo parameter that expects an integer value", :priority
```

Comment:
This is never defined as a shared example group, which is causing the tests to fail early. Is this supposed to be "`a yumrepo parameter that can be an integer`"?

Reply:
Thanks for pointing the error here! I couldn't get what was wrong from the message in travis... Fixed.